script/run_tests

```diff
@@ -51,6 +51,9 @@ def RunTests(binary_tests):
       if disabled_tests and test_name in disabled_tests and disabled_tests[test_name]:
         exclusion_filter = '-' + ':'.join(disabled_tests[test_name])
         test_args.append('--gtest_filter=%s' % exclusion_filter)
+      # Add reporting path if enabled
+      if generate_report:
+        test_args.append("--gtest_output=xml:" + os.path.join(SOURCE_ROOT, 'test_reports', test_name + os.path.extsep + 'xml'))
```

Comment:
```python
os.path.join(SOURCE_ROOT, 'test_reports', test_name + os.path.extsep + 'xml')
```
@nitsakh Can you please move it to a function? Looks really complicated inline )

Reply:
It doesn't do much. I can change `os.path.extsep + 'xml'` to directly `.xml` is that'll make it simpler. Extension separators are same on Win, Mac and Linux anyway.

Reply:
The problem is not that it's doing to much.
It's that I have to read it through to understand what's going on there.
Having `test_args.append("--gtest_output=xml:" + get_output_file_path(test_name))` is much easier to read.

Yeah, I'm kinda nitpicking here, sorry =/